CoreWidgetProvider/Helpers/ChartHelper.cs

```diff
@@ -19,8 +19,13 @@ public enum ChartType
 
     public const int ChartHeight = 86;
     public const int ChartWidth = 264;
+    private const int NumOfYGridlines = 4;

```

Comment:
Why these numbers?  What is magical about them?

Reply:
We have a bunch of pending questions for Design on what they want here, but since the graphs are so short, 4 lines (one every 20%) seems like it might be the way to go (as opposed to every 10% like in task manager).